components/01-atoms/videos/video-embed.js

```diff
@@ -0,0 +1,27 @@
+Drupal.behaviors.responsiveEmbeddedVideos = {
```

Comment:
What is this javascript for? I think we should be able to do responsive videos with css only.

Reply:
Never mind.

Reply:
We figured out what it's doing during our FEPG. It would be great to have this documented somewhere so that anyone coming to this cold knows it's utility and use-case.